pennathletics/athletes.py

```diff
@@ -17,6 +17,28 @@ def get_roster(sport, year):
             parse.parse_hometown(person[6]))
         )
     return roster
-        
     
-# print(get_roster('M_Basketball',2015))
\ No newline at end of file
+def get_player(sport, year, name = "", no = -1):
```

Comment:
Today I'll working on setting up automatic CI testing to check for this stuff. Also yeah linters.